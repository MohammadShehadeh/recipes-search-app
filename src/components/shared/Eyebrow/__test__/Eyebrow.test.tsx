import renderer from 'react-test-renderer';

import { Eyebrow } from '..';

describe('Eyebrow', () => {
  it('should render "contained" to match snapshot', () => {
    const tree = renderer.create(<Eyebrow label="Contained Eyebrow" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render "outlined" to match snapshot', () => {
    const tree = renderer
      .create(<Eyebrow variant="outlined" label="Contained Eyebrow" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
