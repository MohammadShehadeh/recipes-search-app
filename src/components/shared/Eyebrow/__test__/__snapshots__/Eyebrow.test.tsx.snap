// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Eyebrow should render "contained" to match snapshot 1`] = `
<span
  className="eyebrow contained"
>
  <span
    className="label"
  >
    Contained Eyebrow
  </span>
</span>
`;

exports[`Eyebrow should render "outlined" to match snapshot 1`] = `
<span
  className="eyebrow outlined"
>
  <span
    className="label"
  >
    Contained Eyebrow
  </span>
</span>
`;
